CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
   emitted in source order as much as possible, instead of always being emitted
   after the CSS of all module dependencies.
 
+* All functions defined in CSS Values and Units 4 are now parsed as calculation objects:
+round(), mod(), rem(), sin(), cos(), tan(), asin(), acos(), atan(), atan2(), pow(), sqrt(), hypot(), log(), exp(), abs(), and sign().
```

Comment:
Style nit: reflow the paragraph at 80 characters and code-format function names.